frontend/src/components/Home.vue

```diff
@@ -68,8 +73,8 @@
                   <tr v-if="el.date">
                     <td>{{ el.date }}</td>
```

Comment:
The new date format (dash instead of dot as separator) allows the dates to wrap into multiple lines. One way to fix this could be to have a `white-space:nowrap;` rule on the relevant elements.

<img width="590" alt="image" src="https://user-images.githubusercontent.com/423498/126960315-7a10596c-f41f-47b9-b377-a853d644ec92.png">

<img width="467" alt="image" src="https://user-images.githubusercontent.com/423498/126960655-d4dca71c-4980-46bc-bd5c-40594d19e3df.png">

Reply:
By the way, I didn't notice this problem in Firefox, so I had to change to Chrome to trigger it.